src/app.js

```diff
@@ -4,7 +4,9 @@ const cors = require('cors')
 const favicon = require('express-favicon');
 const logger = require('morgan');
 
-const mainRouter = require('./routes/mainRouter.js');
+const mainRouter = require('./routes/mainRouter');
```

Comment:
you can have an index file in the routes folder and move all route related info into it

Reply:
Will do. @Austine105 Thank you